lib/commands/element.js

```diff
@@ -250,6 +250,19 @@ const commands = {
     const atomsElement = this.getAtomsElement(el);
     await this.executeAtom('click', [atomsElement]);
     await this.executeAtom('type', [atomsElement, value]);
+    if (this.opts.triggerInputEventAfterSendkeys) {
```

Comment:
I assume ideally the original selenium atom must be fixed instead
Adding of workarounds just increases our technical debt and creates more chaos

Reply:
It's not a bug with the atom as far as I can tell. It's to how react handles input events. The atom works fine with normal text fields.

Reply:
then we should probably keep this property enabled by default to stick with how it was behaving before.
Also, it would probably make sense to extract this script into a separate .js file instead of hardcoding it in-place

Reply:
ok. I updated so the function can be an actual JS function. and switched the polarity of the cap.